ee/clickhouse/models/test/test_action.py

```diff
@@ -16,7 +17,10 @@ class MockEvent:
 
 
 def _get_events_for_action(action: Action) -> List[MockEvent]:
-    formatted_query, params = format_action_filter(team_id=action.team_id, action=action, prepend="")
+    hogql_context = HogQLContext()
+    formatted_query, params = format_action_filter(
```

Comment:
Sure, but it's a helper method in a test that already breaks various contracts 🙃. Normally this would be a util/query function somewhere that does all of this internally.

Reply:
😅